### PR TITLE
Electra: full beacon config

### PR DESCRIPTION
**Pull request by prestonvanloon**
**What type of PR is this?**

Other

**What does this PR do? Why is it needed?**

Follow up to #13907. That PR was missing many things.

**Which issues(s) does this PR fix?**

**Other notes for review**

~Not sure why spec test present compliance tests didn't catch this in #13907. That will need some
further investigation.~ Fixed it

Given that the spec present compliance test is not reliable, please check the these constants against the spec presents.

- [mainnet](https://github.com/ethereum/consensus-specs/blob/v1.5.0-alpha.0/presets/mainnet/electra.yaml)
- [minimal](https://github.com/ethereum/consensus-specs/blob/v1.5.0-alpha.0/presets/minimal/electra.yaml)

### CODE REVIEW

File: config/params/config.go [hunk 1]
```diff
@@ -97,12 +99,13 @@ type BeaconChainConfig struct {
...
-	MaxAttestations                  uint64 `yaml:"MAX_ATTESTATIONS" spec:"true"`                     // MaxAttestations defines the maximum allowed attestations in a beacon block.
-	MaxDeposits                      uint64 `yaml:"MAX_DEPOSITS" spec:"true"`                         // MaxDeposits defines the maximum number of validator deposits in a block.
-	MaxVoluntaryExits                uint64 `yaml:"MAX_VOLUNTARY_EXITS" spec:"true"`                  // MaxVoluntaryExits defines the maximum number of validator exits in a block.
-	MaxWithdrawalsPerPayload         uint64 `yaml:"MAX_WITHDRAWALS_PER_PAYLOAD" spec:"true"`          // MaxWithdrawalsPerPayload defines the maximum number of withdrawals in a block.
+	MaxProposerSlashings             uint64 `yaml:"MAX_PROPOSER_SLASHINGS" spec:"true"`      // MaxProposerSlashings defines the maximum number of slashings of proposers possible in a block.
+	MaxAttesterSlashings             uint64 `yaml:"MAX_ATTESTER_SLASHINGS" spec:"true"`      // MaxAttesterSlashings defines the maximum number of casper FFG slashings possible in a block.
```
Comment (james-prysm) [inline config/params/config.go:103]: https://github.com/prysmaticlabs/prysm/pull/13898 has max attester slashings eletra / max attestations electra  fields

Comment (prestonvanloon) [reply config/params/config.go:103]: Thanks, I'll add these

File: beacon-chain/rpc/eth/config/handlers_test.go [hunk 2]
```diff
@@ -158,6 +172,9 @@ func TestGetSpec(t *testing.T) {
 	var dam [4]byte
 	copy(dam[:], []byte{'1', '0', '0', '0'})
 	config.DomainApplicationMask = dam
+	var dc [4]byte
+	copy(dc[:], []byte{'1', '1', '0', '0'})
+	config.DomainConsolidation = dc
```
Comment (kasey) [inline beacon-chain/rpc/eth/config/handlers_test.go:177]: I noticed the spec value is `0x0B000000`, so the first 2 bytes being `1` stuck out as wrong. I think this should be `[]byte[11, 0, 0, 0]`: https://go.dev/play/p/fkwg-mzX8Zb

Comment (prestonvanloon) [reply beacon-chain/rpc/eth/config/handlers_test.go:178]: These are dummy values. This whole test sucks

**Review by kasey — APPROVED** [review]

**Pull request merged**